app/src/views/ERUReadinessUpdateForm/ERUTypeInput/index.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+    Container,
+    InputSection,
+    RadioInput,
+    TextArea,
+} from '@ifrc-go/ui';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+import {
+    type SetValueArg,
+    useFormObject,
+} from '@togglecorp/toggle-form';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+interface ERUType {
+    eruTypeId: string;
+    equipment_readiness: number;
+    people_readiness: number;
+    funding_readiness: number;
+    comments: string;
+    comments_text: string | undefined;
+    eruTypeTitle: string;
+    client_id: string;
+}
+
+interface ERUTypeFormInput {
+    id: string;
+    eruTypes: ERUType[] | undefined;
+    eruNationalSociety: string;
+}
+
+export interface ReadinessOption {
+    value: string;
+    key: number;
+}
+
+type ERUTypeFormFields = NonNullable<ERUTypeFormInput['eruTypes']>[number];
+
+function readinessKeySelector(option: ReadinessOption) {
+    return option.key;
+}
+
+function readinessLabelSelector(option: ReadinessOption) {
+    return option.value;
+}
+const defaultValue: ERUTypeFormFields = {
+    client_id: '-1',
```

Comment:
I know its not super necessary, but would be better if we generate a random key here.